week1_intro/primer/recap_ml.ipynb

```diff
@@ -103,9 +103,9 @@
    "metadata": {},
    "source": [
     "### Part II: Loading data with Pandas\n",
-    "Pandas is a library that helps you load the data, prepare it and perform some lightweight analysis. The god object here is the `pandas.DataFrame` - a 2D table with batteries included. \n",
+    "Pandas is the library that helps you to load the data, prepare it and perform some lightweight analysis. The god object here is the `pandas.DataFrame` - a 2D table with included batteries. \n",
```

Comment:
Should be "a" here, I think. In addition, "batteries included" is a common expression (e.g. https://www.python.org/dev/peps/pep-0206/).